pycqed/instrument_drivers/meta_instrument/LutMans/mw_lutman.py

```diff
@@ -510,61 +538,28 @@ def load_waveforms_onto_AWG_lookuptable(
             regenerate_waveforms (bool): if True calls
                 generate_standard_waveforms before uploading.
             stop_start           (bool): if True stops and starts the AWG.
-
-
-        Because of realtime loading vs the DIO sequencer program the uploading
-        flow for the AWG8 is slightly different.
+            force_load_sequencer_program (bool): if True forces a new compilation
+                and upload of the program on the sequencer.
         """
-        super().load_waveforms_onto_AWG_lookuptable(
-            regenerate_waveforms=regenerate_waveforms,
-            stop_start=stop_start)
         # Uploading the codeword program (again) is needed to link the new
-        # waveforms.
+        # waveforms in case the user has changed the codeword mode.
```

Comment:
Should this functionality (checking if the program has changed) not belong in the HDAWG driver now? 
There seems no reason to have this dedicated function here anymore if this is the case.

Reply:
That's fine, I can remove it.

I just didn't want to mess with your logic in the lutman too much.